tools/scenario-player/scenario_player/exceptions.py

```diff
@@ -30,9 +30,25 @@ class RESTAPIStatusMismatchError(ScenarioError):
     pass
 
 
+class MultipleTaskDefinitions(ScenarioError):
+    """Several tasks were defined in the scenario configuration."""
+
+
+class InvalidScenarioVersion(ScenarioError):
+    pass
+
+
+class UnexpectedScenarioVersion(ScenarioError):
```

Comment:
Looks like this isn't used anywhere.

Reply:
True! Replaced by `InvalidScenarioVersion` and then banished and forgotten. Removed.